.github/workflows/release.yaml

```diff
@@ -15,5 +15,7 @@ jobs:
       - name: Increment version and create PR
         uses: googleapis/release-please-action@v4
         with:
-          token: ${{ secrets.RELEASE_PLEASE_GITHUB_TOKEN }}
-          release-type: python
\ No newline at end of file
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: python
+          skip-github-release: false
+          skip-github-pull-request: true
```

Comment:
Needed to make github actions to skip creating
<img width="1440" alt="Screenshot 2024-05-30 at 1 06 05 PM" src="https://github.com/PsiQ/qref/assets/39071023/6810fda3-694d-470c-8e30-cc40e2904137">
the pull request and letting the bot/app create the PR. This will work now because github actions can create and approve PRs now.